STYLE-GUIDE.md

```diff
@@ -45,6 +45,12 @@ This bit of info is serious. If you missed it, bad things could happen.
 {{% /notes %}}
 ```
 
+## Paragraphs and Line Breaks
+
+When writing paragraphs, keep the entire paragraph on a single line and use "soft wrapping" in your IDE/editor to avoid a horizontal scroll.  By keeping all content on a single line, PR reviewers will be able to use GitHub's suggestion feature, which allows the reviewer to directly suggest a change to the content and allows authors to directly incorporate the suggestion into their PR if desired.
```

Comment:
```suggestion
When writing paragraphs, keep the entire paragraph on a single line and use "soft wrapping" in your IDE/editor to avoid a horizontal scroll. By keeping all content on a single line, PR reviewers will be able to use GitHub's suggestion feature, which allows the reviewer to directly suggest a change to the content and allows authors to directly incorporate the suggestion into their PR if desired.
```